### ISSUE TITLE
Can I add myself to `privileged-requester.yaml` in the PR and get auto approved?

### ISSUE THREAD

**Issue by lawrencegripper** [github/privileged-requester#177]
Is it possible, for example, to edit the [privileged-requester.yaml](https://github.com/github/entitlements/blob/master/config/privileged-requester.yaml) file in the PR where I make my changes so that it includes my user as a privileged requestor?

In a similar issue to :point_down: , could I set `commitVerification` to false in the branch to allow impersonation?

- https://github.com/github/privileged-requester/issues/166

I think these would be mitigated by reading the configuration from the default branch rather than from the PR branch but I'm not sure how to work that with

**Comment by lawrencegripper** [comment]
Yeah this is the route we try and use on Heaven when reading configuration that is enforcing/permissions related, we try and always read from the main branch. It's not always possible tho

**Comment by GrantBirki** [comment]
IMO we need to update this Action with a new major version (breaking change) that ensures all config options related to security live on the `main` branch outside of the Action's config
